packages/components/src/DynamicModal.tsx

```diff
@@ -38,6 +53,10 @@ export const useDynamicModal = () => {
   const [modalProps, setModalProps] = useState<
     ThemingProps & { onClose: () => void | Promise<void> }
   >(defaultModalProps);
+  const motionPreset = useBreakpointValue({
+    base: "slideInBottom",
+    lg: (localStorage.getItem("chakra-modal-motion-preset") as any) || undefined,
```

Comment:
please use the same value for both. now it should work fine with e2e tests. and please remove the `chakra-modal-motion-preset` leftovers